packages/editor-ui/src/stores/settings.ts

```diff
@@ -249,6 +263,26 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, {
 			const rootStore = useRootStore();
 			await deleteApiKey(rootStore.getRestApiContext);
 		},
+		async getLdapConfig() {
+			const rootStore = useRootStore();
+			return await getLdapConfig(rootStore.getRestApiContext);
```

Comment:
we should setup a new store for all things LDAP, and then use that for rendering the UI. right now we are making unnecessary requests to the backend even when have all the data we need to render. 
Also, when we save the ldap config, right now if the backend replaces any empty values with default values, that does not reflect in the UI because this code isn't using a state for rendering the form.